docs/decisions/0003-initial-api-specification.rst

```diff
@@ -93,3 +93,145 @@ Permissions
 enterprise_learner
   Should only create the transaction if the requesting user has implicit (JWT) or explicit (DB-defined)
   ``enterprise_learner`` role assigned.
+
+
+CRUD operations on SubsidyAccessPolicy
+======================================
+
+GET **/api/v1/admin/policy/**
+
+List all policies linked to a group.
+
+Inputs
+------
+
+- ``group_uuid`` (Query Param, required): The uuid of the customer. For now it will be an enterprise customer uuid.
+
+Outputs
+-------
+A sample response can be seen below.
+
+::
+
+   [
+        {
+            "uuid": "d0cd25f9-0b73-49c3-8299-dc9751a12ef5",
+            "policy_type": "PerLearnerEnrollmentCreditAccessPolicy",
+            "description": "sdad",
+            "active": true,
+            "group_uuid": "214cf999-5964-4a2e-afa8-c62461558211",
```

Comment:
we probably want `enterprise_customer_uuid` here, `group` in this context will have a specific, important meaning in the future.